File: Toy-pro/src/Fizz-Buzz/FizzBuzz_RC.java
/* FizzBuzz_RC.java*/

public class FizzBuzz_RC {

	public static void main(String[] args) {
	
		FizzBuzz_RC fb;
		fb = new FizzBuzz_RC();
		fb.fizzbuzz(100);
	}
	
	void fizzbuzz(int n){
	
	if(n > 1)
		fizzbuzz(n-1);

	if(n % 3 == 0 && n % 5 == 0)
		System.out.println("Fizz Buzz");
	
	else if(n % 3 == 0)
		System.out.println("Fizz");

	else if(n % 5 == 0)
		System.out.println("Buzz");

	else
		System.out.println(n);

	}
}

File: Toy-pro/src/Fizz-Buzz/fizzbuzz_rc.c
/*Fizz-Buzz_rc.c*/

#include<stdio.h>

void fizz_buzz(int n){
	
	if(n > 1)
		fizz_buzz(n-1);

	if (n % 3 == 0 && n % 5 == 0)
		printf("Fizz Buzz\n");

	else if (n % 3 == 0)
		printf("Fizz\n");

	else if (n % 5 == 0)
		printf("Buzz\n");

	else
		printf("%d\n",n);
}

int main(void){

	fizz_buzz(100);

	return 0;
}

